src/main/java/com/example/clone/model/Comment.java

```diff
@@ -0,0 +1,33 @@
+package com.example.clone.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
```

Comment:
Remove lombok from the project.
Lombok is great, but I want you to know how things are working under the hood. Lombok can be added later.